Week2/homework/round.js

```diff
@@ -0,0 +1,36 @@
+//5.1 Declare a variable z and assign the number 7.25 to it.
+const z= 7.25;
+
+
+
+//5.2 console.log z.
+console.log(z);
+
+
+
+//5.3 Declare another variable a that has the value of z but rounded to the nearest integer.
+const a=  Math.round(z);
+
+
+
+//5.4 console.log a.
+console.log(a);
+
+
+
+
+//5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
+ let h;
+ if (z > a)
+
+{
+   
+    h=z;
+
+}
```

Comment:
Ah, this solution is not complete, please check it again.  Now you only check if `z` is higher than `a` and then setting `z` to the highest value. But what if `a` is higher than `z`?

Reply:
you're right, i have changed it by using Math.max().